meson.build

```diff
@@ -1,8 +1,15 @@
-project('dxvk', ['c', 'cpp'], default_options : ['cpp_std=c++1z'])
+project('dxvk', ['c', 'cpp'])
 
 cpu_family = target_machine.cpu_family()
 
 dxvk_compiler = meson.get_compiler('cpp')
+if dxvk_compiler.get_id() == 'msvc'
+  add_global_arguments('-DNOMINMAX', language : 'cpp')
+  dxvk_cpp_std='c++17'
+else
+  dxvk_cpp_std='c++1z'
```

Comment:
leaving this as is might break MSVC if we starting using new features. might be safer to use c++17 everywhere.

Reply:
`std=c++17` breaks GCC 6 *now*. This was changed from using c++17 everywhere for that specific reason.

Reply:
MSVC is behind GCC in supporting some c++17 features anyway, so we already have to be careful not to break it. Maybe use c++latest instead for MSVC?

Reply:
fixed